src/pet.js

```diff
@@ -0,0 +1,73 @@
+const max_fitness = 10;
+const empty_belly = 0;
+const grow_old = 1;
+
+function Pet (name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = empty_belly;
+    this.fitness = max_fitness;
+    this.child = [];
+    };
+
+    Pet.prototype = {
+        get isAlive(){
+            return this.age < 30 && this.hunger < 10 && this.fitness > 0
+        }
+    };
+    
+    Pet.prototype.growUp = function (){
+        if (!this.isAlive){
+            throw new Error('Your pet is no longer alive')
+        }
+        this.age += grow_old;
+    };
+    Pet.prototype.getHungry = function (){ 
```

Comment:
You should always leave a clear line between each of your functions
```
};

Pet.prototype.getHungry = function () {
```
Also, there should be a space between the closing ) bracket and the opening { in the first line of all your functions